NAMESPACE

```diff
@@ -121,7 +121,6 @@ export(setDefaultNumericFormat)
 export(setDefaultPlotFormat)
 export(setDefaultStairstep)
 export(setDefaultTheme)
-export(setDefaultThemeFromJson)

```

Comment:
`setDefaultThemeFromJson` removed because the function is available in the **tlf**?

Reply:
I removed it because I indeed thought so, but actually it is not the case.
So, I'll put it back on RE